filler.go

```diff
@@ -177,6 +187,35 @@ func (f *formFiller) fill(ctx reflect.Value) error {
 	return nil
 }
 
+func (m *multiFormFiller) fill(ctx reflect.Value) error {
+	for k, idx := range m.multiFormParams {
+		// left shift 20 offset means MB
+		err := m.req.ParseMultipartForm(int64(m.limit[k]) << 20)
+		if err != nil {
+			return err
+		}
+		file, _, err := m.req.FormFile(k)
+		if err != nil {
+			return err
+		}
```

Comment:
multi-form not always is file, it might be value too.

Reply:
Then tag should be `multiformfile:"key"`? And distinct with `multiform`

Reply:
Or just `file:"key"`?

Reply:
I prefer `multiform`, since I expected the tag should be Content-Type of http.

Reply:
Do you mean `multiform` for all cases? Probably we could use `multiform:"key" file:"yes"` to distinct them?

Reply:
I think `multiform:"key" file:"yes"` is good.